pandas/io/parsers.py

```diff
@@ -320,26 +338,10 @@
     tool, ``csv.Sniffer``. In addition, separators longer than 1 character and
     different from ``'\s+'`` will be interpreted as regular expressions and
     will also force the use of the Python parsing engine. Note that regex
-    delimiters are prone to ignoring quoted data. Regex example: ``'\r\t'``
+    delimiters are prone to ignoring quoted data. Regex example: ``'\r\t'``.
 delimiter : str, default ``None``
-    Alternative argument name for sep."""
-
-_read_csv_doc = """
-Read CSV (comma-separated) file into DataFrame
-
-%s
-""" % (_parser_params % (_sep_doc.format(default="','"), _engine_doc))
-
-_read_table_doc = """
-
-.. deprecated:: 0.24.0
-   Use :func:`pandas.read_csv` instead, passing ``sep='\t'`` if necessary.
-
-Read general delimited file into DataFrame
-
-%s
-""" % (_parser_params % (_sep_doc.format(default="\\t (tab-stop)"),
-                         _engine_doc))
+    Alias for sep.
+    """
 
 _fwf_widths = """\
```

Comment:
Changes look great.

I just think this is not being used anymore. Can you check please? And also run the `validate_docstrings.py` on the 3 docstrings edited here, to see that everything is ok? I guess the script should be reporting that these params are missing in the docstring.

Reply:
Yes. It is not used anymore. I run  both `validate_docstrings.py` and `python make.py --single`, and they all look fine except `{**kwds} error` from `read_fwf`.

Reply:
That sounds good. There is something that just by the code I don't understand. I see the signature of `read_fwf` is `def read_fwf(filepath_or_buffer, colspecs='infer', widths=None, **kwds):`, but we're using the docstring in `_parser_doc` that has lots of other parameters like `header`... That doesn't seem correct. Is the script not complaining of unknown parameters?

Or I'm missing something?

Reply:
I forgot to add `_fwf_widths` back to `read_fwf`. But the script is still complaining of  unknow parameters. 
```
Parameters {**kwds} not documented
        Unknown parameters {decimal, float_precision, ...
```
I thought there will be a fix in the test to accommodate `{**kwds}`.